projects/packages/search/src/dashboard/components/dashboard/pages/loading.jsx

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Should this be outside the `pages` folder, given that it's more of a component than a full page?

Reply:
Good point 👍 addressed in 28b81ab